nix/test-nixos-configuration.nix

```diff
@@ -6,9 +6,11 @@
     memorySize = 8192;
     diskSize = 100000;
     forwardPorts = [
+      # SSH
```

Comment:
Forwarding to 22 seems like a bad idea, it'd mean you couldn't ssh into the host machine, right?

Reply:
`host.port` is set to 2222, isn't it the port on the host machine?